packages/api/apollo-server/mutations/index.ts

```diff
@@ -0,0 +1,7 @@
+import userMutation from './users'
+
+export default {
+  Mutation: {
+    userMutation
```

Comment:
It's not clear what this mutation does. Does it edit the user? delete the user? add a user?